etcd/tasks.go

```diff
@@ -2,26 +2,44 @@ package etcd
 
 import (
 	"github.com/giantswarm/microerror"
+	"time"
 )
 
 func FullBackup(b BackupInterface) error {
 	var err error
 
 	version := b.Version()
 
+	start := time.Now()
+
 	err = b.Create()
 	if err != nil {
 		return microerror.Maskf(err, "Etcd %s creation failed: %s", version, err)
 	}
 
+	creationTime := time.Since(start).Nanoseconds() / 1e6
```

Comment:
why not use `time.Since(start).Seconds()` instead of this magic?  Magic constants are usually discouraged

Reply:
make sense, done